.vscode/settings.json

```diff
@@ -21,5 +21,8 @@
     "editor.formatOnSave": true
   },
   "typescript.tsdk": "node_modules/typescript/lib",
-  "editor.rulers": [100]
+  "editor.rulers": [100],
+  "bazel.buildifierExecutable": "node_modules/.bin/buildifier",
+  "bazel.buildifierFixOnFormat": true,
+  "bazel.executable": "node_modules/.bin/bazel"
```

Comment:
This might break on Windows, but I'm not sure. Worth giving a shot.

Reply:
I think it might work, the things in node_modules/.bin appear to be nodejs executables that delegate to the appropriate native binary.

Reply:
On Windows, the files are often `.cmd`. E.g. `node_modules/.bin/bazel.cmd`

Reply:
It's odd that they would make a nodejs program to delegate to different native implementations and then not give it the same name across environments, kind of defeats the point I would think... 🤷‍♂️